### ISSUE TITLE
Server should respond on port 443 (HTTPS/TLS)

### ISSUE THREAD

**Issue by Fry-kun** [AdAway/AdAway#715]
Some ads are on HTTPS servers, e.g. google-analytics:

Network compressed POST connection error: java.net.ConnectException: failed to connect to ssl.google-analytics.com/127.0.0.1 (port 443) after 60000ms: isConnected failed: ECONNREFUSED (Connection refused)

Instead of 60s timeout, the local webserver could respond in milliseconds with some reply (no such server name?), bypassing the problem

**Comment by 0-kaladin** [comment]
not sure why you have a 60s timeout.  I did tests on 443 on the webserver, read this: https://github.com/Free-Software-for-Android/AdAway/issues/667

with or without the web server, the response was immediate, just different.  so there was no gain in running another instance of the webserver on port 443.

**Comment by Fry-kun** [comment]
I'm guessing different apps react differently to the "server is not up" response vs. "certificate is invalid" response.
I got a small boost in one app when I redirected settings.crashlytics.com to a private server that answers on port 443 with an invalid certificate
logcat indicated that with regular localhost redirect, this connection times out after 10sec but with an invalid cert, it fails immediately.
